packages/patternfly-4/react-table/src/components/Table/Body.js

```diff
@@ -19,15 +19,17 @@ const defaultProps = {
 };
 
 class ContextBody extends React.Component {
-  onRow = (row, props) => {
+  onRow = (row, rowProps) => {
     const { onRowClick } = this.props;
     return {
-      isExpanded: row.isExpanded,
-      isOpen: row.isOpen,
-      onClick: event => {
-        if (event.target.tagName !== 'INPUT' && event.target.tagName !== 'BUTTON') {
-          onRowClick(event, row, props);
-        }
+      row,
+      rowProps,
+      onMouseDown: event => {
+        const computedData = {
```

Comment:
inline edit needs input events. I can send back computedData which are a bit easier to consume and leave the decision logic on consumer. IMO this is the best what can be done. Does somebody have a better idea?